libs/ledgerjs/packages/context-module/src/external-plugin/data/HttpExternalPluginDataSource.ts

```diff
@@ -0,0 +1,42 @@
+import axios from "axios";
+import { ExternalPluginDataSource, GetDappInfos } from "./ExternalPluginDataSource";
+import { DappResponse } from "./DappResponse";
+import { DappInfos } from "../model/DappInfos";
+import { SelectorDetails } from "../model/SelectorDetails";
+
+export class HttpExternalPluginDataSource implements ExternalPluginDataSource {
+  constructor() {}
+
+  async getDappInfos({ chainId, address, selector }: GetDappInfos): Promise<DappInfos | undefined> {
+    const dappInfos = await axios.request<DappResponse[]>({
+      method: "GET",
+      url: "https://crypto-assets-service.api.ledger.com/v1/dapps",
+      params: { output: "b2c,b2c_signatures,abis", chain_id: chainId, contracts: address },
```

Comment:
Little request, could you add `X-Ledger-Client-Version` header to all your requests ? We use it on backend side to properly categorize clients:
![image](https://github.com/LedgerHQ/ledger-live/assets/102984500/c2603f0c-e39c-414a-9e5c-94d53a5258fa)

Expected syntax is `X-Ledger-Client-Version: <app>/<version>` (we don't care much about version, eg `X-Ledger-Client-Version: device-sdk/0.0.0` is fine)